packages/components/src/Form/docs/nativeValidation.tsx

```diff
@@ -0,0 +1,22 @@
+import { Form, TextField, Label, Div, ErrorMessage, Button, ButtonGroup } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Div margin="stack-lg">
+            <Form validationBehavior="native">
+                <TextField
+                    name="email"
+                    type="email"
+                    isRequired
+                >
+                    <Label>Email</Label>
+                    <ErrorMessage>This field is required</ErrorMessage>
```

Comment:
Might not be a huge deal, but when you type an invalid email, it also shows the required error message.
Is there anyway to have two separate error messages for this?

Reply:
maybe if you don't provide a message like here and use the built in option https://react-spectrum.adobe.com/react-aria/forms.html#built-in-validation

Reply:
or you can customize them https://react-spectrum.adobe.com/react-aria/forms.html#customizing-error-messages